keyboards/keyquest/enclave/info.json

```
@@ -0,0 +1,22 @@
+{
+    "manufacturer": "keyquest",
+    "keyboard_name": "Enclave-1",
+    "maintainer": "the keyquest team",
+    "layouts": {
+        "LAYOUT_macro_3x3": {
```

This should probably just be `LAYOUT`.  Eg
```suggestion
        "LAYOUT": {
```

Though, changing it here would require changing it anywhere else it's being used, too